README.md

```diff
@@ -2,40 +2,30 @@
 
 # Project Name
 
-Mobile Portofolio Microverse
+My Portofolio Project
 
-This is our mobile portofolio project in Microverse's module one. In this project, things like setting up html and css liters, GitHub Actions, and following GitHub flow are covered.
+This is portofolio project. In this project, things like setting up html and css,liters, GitHub Actions, and following GitHub flow are covered.The Template i used is provided by Microverse is here:https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471
 
 
 ## Built With
 
-Html and CSS ,Linters
+ -Html
+ -CSS 
+ -Linters
 
-## Live Demo (if available)
+## Live Demo 
 
-[Live Demo Link](https://livedemo.com)
+https://lul-abdifan.github.io/Mobile_Portofolio/
```

Comment:
- [ ] **[OPTIONAL]**: Great job with your deployment! :partying_face:, however, I would suggest :bulb:  making your link easily accessible with a click by implementing markdown to wrap the URL of your live demo like so:

`[Live Demo](URL of your project)`

In this way, you can make sure that anyone can access your portfolio easily by just clicking :100:

Reply:
You showed me which needs improvement .I will fix it before I merge.Thank you for your careful feedback.